src/js/tabs/firmware_flasher.js

```diff
@@ -574,12 +574,8 @@ firmware_flasher.initialize = function (callback) {
             self.isFlashing = false;
         }
 
-        let result = getConfig('erase_chip');
-        if (result.erase_chip) {
-            $('input.erase_chip').prop('checked', true);
-        } else {
-            $('input.erase_chip').prop('checked', false);
```

Comment:
here it is disabled by default if key is not present or false

Reply:
everything "fixed" by only setting this `true` always; nothing more needed.
and thusly, pilots can disable manually per configurator execution.
it will automatically re-enable upon re-launch.

no reason this should be saved locally. otherwise, pilots will break their own quads repeatedly.

i did this in "other project" in 2020, no complaintive feedback by pilots.

Reply:
not done cleanly, but this: https://github.com/emuflight/EmuConfigurator/pull/180/files

Reply:
Yes this PR has same outcome as in yours it was better to remove the condition as both paths leads to true but we don't need to nuke the setting on startup :)